.gitlab-ci.yml

```diff
@@ -0,0 +1,35 @@
+image: espressomd/buildenv-espresso-ubuntu-cuda:python
+build_and_test:
+  stage: build
+  script:
+    - bash maintainer/cuda_build.sh
+
+
+sphinx_documentation:
+  stage: deploy
+  only:
+    - doc
```

Comment:
That's internal stuff, happening on the endpoint site. We only want to update the doc if pushes are received. Therefore I force push to doc branch for pull requests.